packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -849,19 +893,35 @@ contract Deploy is Deployer {
 
     /// @notice Deploy the AnchorStateRegistry
     function deployAnchorStateRegistry() public broadcast returns (address addr_) {
-        IAnchorStateRegistry anchorStateRegistry =
-            IAnchorStateRegistry(_deploy("AnchorStateRegistry", abi.encode(mustGetAddress("DisputeGameFactoryProxy"))));
+        IAnchorStateRegistry anchorStateRegistry = IAnchorStateRegistry(
+            DeployUtils.create2AndSave({
+                _save: this,
+                _name: "AnchorStateRegistry",
```

Comment:
Minor nit but for aesthetics let's put the `_salt` parameter underneath the `_save` parameter, will make the code look a bit nicer

Reply:
Worth applying everywhere in this file